docs/install.md

```diff
@@ -1,20 +1,22 @@
-Production Setup
-================
+Fluo Install Instructions
+=========================
 
-Below are instructions for running Fluo in a production environment where Accumulo,
-Hadoop & Zookeeper are installed and running.  If you want to avoid setting up
-these dependencies, consider using [fluo-dev]. 
+Install instructions for running Fluo on machine or cluster where Accumulo, Hadoop,
+and Zookeeper are installed and running.  If you want to avoid setting up these
+dependencies, consider using [fluo-dev] or [Zetten].
 
 Requirements
 ------------
 
-Before you install Fluo, you will need the following installed and running on
+Before you install Fluo, the following software must be installed and running on
 your local machine or cluster:
 
-* [Accumulo][Accumulo] (version 1.6+)
-* [Hadoop][Hadoop] (version 2.6+)
-* [Zookeeper]
-* [Java][Java] (version 8+)
+| Software             | Recommended Version | Minimum Version |
+|----------------------|---------------------|-----------------|
+| [Accumulo][Accumulo] | 1.7.2               | 1.6.1           |
```

Comment:
This is just left over from the previous commit.  It was done before the to prevent `(version 1.6+)` from being included in link.  Now that it's in a table. I can make it just `[Accumulo]`